### ISSUE TITLE
Second test does not run on Mac

### ISSUE THREAD

**Issue by ozawakei** [DevExpress/testcafe#6833]
### What is your Scenario?

The first test has been completed successfully.
However, when I run the same test again, Testcafe does not start the test.

### What is the Current behavior?

The browser starts, but the test page does not load.

### What is the Expected behavior?

The same test ends again.

### What is your public website URL? (or attach your complete example)

I have tried https://www.google.com/

### What is your TestCafe test code?

```
import "testcafe";

fixture("simple test").page("https://www.google.com/")

test("test a", async (t) => {
});
```

### Your complete test report

Using locally installed version of TestCafe.
 Running tests in:
 - Chrome 97.0.4692.99 / macOS 10.15.7

 simple test
ERROR The Chrome 97.0.4692.99 / macOS 10.15.7 browser disconnected. If you did not close the browser yourself, browser performance or network issues may be at fault.

### Screenshots

<img width="1191" alt="testcafe" src="https://user-images.githubusercontent.com/93967925/150913498-baf435cf-0bc9-4825-85a7-346cb4410d61.png">

### Steps to Reproduce

1. Start Testcafe and make sure that Testcafe is terminated successfully.
2. Start Testcafe again with the same test code.
3. The browser starts, but the test page is not displayed.

### TestCafe version

1.18.2

### Node.js version

v12.9.0

### Command-line arguments

testcafe chrome tests/simple.js

### Browser name(s) and version(s)

chrome 97.0.4692.99

### Platform(s) and version(s)

macOS 10.15.7

### Other

Reload the page and the test will proceed.

**Comment by AlexanderMoiseev** [comment]
Hello,

Unfortunately, we could not reproduce this behavior. Please send us your system overview as in the attached screenshot so we can investigate it further.

![image](https://user-images.githubusercontent.com/1498953/150957100-095435f0-a20e-4300-a58f-28a2284076eb.png)

Also, please try running your example on a different browser and let us know if the problem persists in this case.

**Comment by ozawakei** [comment]
Hi.
This is my system overview.
<img width="519" alt="スクリーンショット 2022-01-25 19 49 49" src="https://user-images.githubusercontent.com/93967925/150963220-5c6f6c99-844b-4a5c-bd06-4c3d4732d1a6.png">

I have tried safari/firefox, but the problem persists...

**Comment by AlexanderMoiseev** [comment]
Hello,

We could not reproduce this behavior even with your configuration.

The issue can be related to third-party software. Please follow instructions outlined in the following [issue](https://github.com/DevExpress/testcafe/issues/6298#issuecomment-879103035). As a workaround, you can try setting the --hostname flag to localhost, e.g., 

`testcafe chrome tests/simple.js --hostname localhost`

**Comment by ozawakei** [comment]
I have tried what you suggested,  but 
--hostname localhost -> not working
- Created a node server -> the page opens with my IP address

Btw, I also investigated the state of the page in Developer Tools:
- document.readyState is 'loading'
- document.body is 'null'

and the state of the Network tab is as below:
<img width="628" alt="スクリーンショット 2022-01-31 11 54 09" src="https://user-images.githubusercontent.com/93967925/151759231-d37de59d-4387-4ff0-80e5-e7e6557190f5.png">

**Comment by VasilyStrelyaev** [comment]
Thank you for the additional information.

Please also try a different Node.js version (for instance, the latest LTS version 16.13) and see if this fixes the issue. If the issue persists, provide the following details:

1. Clarify if you see the same behavior with `--hostname localhost` or you are unable to run tests with this option at all?
2. Do you have any errors in the console?
3. After a problematic run, when are you able to run the same test successfully again (as during the first attempt)? After a certain time passes, after you restart the Mac, or after you run a different test?
4. Collect and send us a browser's HAR report. To do this, run TestCafe (the second attempt, when the issue manifests) in the remote mode (`testcafe remote tests/simple.js`) and copy the provided URL from the console. In Chrome, open the Network tab in the Developer tools, paste the URL in the address bar, then [collect and share an HAR report](https://support.zendesk.com/hc/en-us/articles/204410413-Generating-a-HAR-file-for-troubleshooting).

**Comment by GOPIRAJU11** [comment]
@ozawakei did u find the solution?

**Comment by ozawakei** [comment]
Thank you for the follow up. 

@GOPIRAJU11 
I may have solved the issue by rebooting my Mac (unchecking "Reopen windows when logging back in").

@AlexanderMoiseev  and @VasilyStrelyaev
I appreciated and thank you for the support.

I have also tried 
- Node.js v16.3 
- "--hostname localhost" option.
I checked the developer console, but there was no error but the loading status of some js was "pending".
